src/ipc-wrapper/server.js

```diff
@@ -0,0 +1,55 @@
+// @ts-check
+import { createServer } from 'rpc-reflector'
+import { SubChannel } from './sub-channel.js'
+
+/**
+ * @param {import('../mapeo-manager.js').MapeoManager} manager
+ * @param {import('rpc-reflector/server.js').MessagePortLike} messagePort
+ */
+export function createMapeoServer(manager, messagePort) {
+  // TODO: LRU? project.close() after time without use?
+  /** @type {Map<string, { close: () => void, project: import('../mapeo-project.js').MapeoProject }>}*/
+  const existingProjectServers = new Map()
+
+  const managerChannel = new SubChannel(messagePort, '@@manager')
+
+  const managerServer = createServer(manager, managerChannel)
+
+  messagePort.on('message', handleMessage)
+
+  return {
+    close() {
```

Comment:
does this need to emit some "closing" event so that projects on the client can do additional handling when it comes to closing?